src/Toggle/GetGroupTogglesForRelease.php

```diff
@@ -49,6 +49,7 @@ public function execute(Request $request)
         /** @var Toggle $toggle */
         foreach($toggles as $toggle) {
             $json[] = [
+                'id' => $toggle->getId(),
```

Comment:
it would be convenient if the api used the same format as GetTogglesForRelease

i.e. id, name, description, url